docker-compose.yml

```diff
@@ -69,6 +69,14 @@ services:
     restart: on-failure
     depends_on:
       - zookeeper
+  watcher:
+    image: kafka-watcher:${MON_KAFKA_WATCHER_VERSION}
+    environment:
```

Comment:
Does it have any logs? If yes then add `LOGSTASH_FIELDS: "service=kafka-watcher"`